src/project-manager.ts

```diff
@@ -698,6 +694,11 @@ export interface FileSystemNode {
 export class InMemoryFileSystem implements ts.ParseConfigHost, ts.ModuleResolutionHost {
 
 	/**
+	 * Contains in-memory map (path => content) for all TypeScript libraries
+	 */
+	private typeScriptLibraries = new Map<string, string>();
```

Comment:
Idea: could we maybe just save the typescript libraries in the normal `files` Map, but with a custom URI scheme like `tslib:`? That's how I did it in PHP: https://sourcegraph.com/github.com/felixfbecker/php-language-server/-/blob/src/ComposerScripts.php#L55:1-56:1